### PR TITLE
Code Climate test reporter

### PR DESCRIPTION
**Pull request by sgerrand**
:information_desk_person: These changes add Code Climate's test reporter to the build process. Also added badges for Code Climate's GPA and test coverage scores.

### CODE REVIEW

**Comment by sgerrand** [comment]
@timstott, @dalizard: Would you please review these changes.

**Comment by timstott** [comment]
:+1: LGTM. Thanks for doing this :blush:

**Comment by sgerrand** [comment]
:gift_heart: :family:

File: fluent-plugin-graylog.gemspec [hunk 1]
```diff
@@ -18,6 +17,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
+  spec.required_ruby_version = '>= 1.9.3'
```
Comment (sldblog) [inline fluent-plugin-graylog.gemspec]: Nitpick: this version is end of lifed. Do we want to support it?

Comment (sgerrand) [inline fluent-plugin-graylog.gemspec]: @sldblog: There was _no_ version restriction before. :wink:

Comment (sldblog) [inline fluent-plugin-graylog.gemspec]: Yes, but still, do we have to work on 1.9?

Comment (sgerrand) [inline fluent-plugin-graylog.gemspec]: @sldblog: Given that this is a plugin for a _system_ software package, I'd assume that the MRI Ruby executable could be anything from v1.9.3 up.

Comment (sldblog) [inline fluent-plugin-graylog.gemspec]: Right, so supporting older distributions. Got it. Thank you! :bow:

**Pull request merged**